core/src/test/java/com/graphhopper/routing/AlternativeRouteCHTest.java

```diff
@@ -57,23 +58,23 @@ public GraphHopperStorage createTestGraph(EncodingManager tmpEM) {
         // has to be locally-shortest to be considered.
         // So we get all three alternatives.
 
-        graph.edge(5, 6, 10000, true);
-        graph.edge(6, 3, 10000, true);
-        graph.edge(3, 4, 10000, true);
-        graph.edge(4, 10, 10000, true);
+        GHUtility.setProperties(graph.edge(5, 6).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(6, 3).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(3, 4).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(4, 10).setDistance(10000), carFE, 60, true, true);
 
-        graph.edge(6, 7, 10000, true);
-        graph.edge(7, 8, 10000, true);
-        graph.edge(8, 4, 10000, true);
+        GHUtility.setProperties(graph.edge(6, 7).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(7, 8).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(8, 4).setDistance(10000), carFE, 60, true, true);
 
-        graph.edge(5, 1, 10000, true);
-        graph.edge(1, 9, 10000, true);
-        graph.edge(9, 2, 10000, true);
-        graph.edge(2, 3, 10000, true);
+        GHUtility.setProperties(graph.edge(5, 1).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(1, 9).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(9, 2).setDistance(10000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(2, 3).setDistance(10000), carFE, 60, true, true);
 
-        graph.edge(4, 11, 9000, true);
-        graph.edge(11, 12, 9000, true);
-        graph.edge(12, 10, 10000, true);
+        GHUtility.setProperties(graph.edge(4, 11).setDistance(9000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(11, 12).setDistance(9000), carFE, 60, true, true);
+        GHUtility.setProperties(graph.edge(12, 10).setDistance(10000), carFE, 60, true, true);
```

Comment:
Hm I think for tests like this `graph.edge(b, b, dist, reverse)` was quite useful? Can we not keep it somehow?

Reply:
At least I would add `GHUtility.edge(graph, a, b, dist, bidir)`?

Reply:
It is not easy as the information which encoder should be used is required.

Reply:
Ah true we would need `GHUtility.edge(graphHopperStorage...)` then. Hm, maybe we do not need this after all when we tweak the GHUtility method(s) a bit.